_dev/docker/docker-compose.yml

```diff
@@ -0,0 +1,166 @@
+version: "3.7"
+services:
+  # 123done:
```

Comment:
Why is 123done commented out?

Reply:
Because I don't have it working yet in the compose environment. The site loads, but logging in fails because `config-local.json` doesn't work. 123done isn't configurable with environment variables so it'll need a new json file.

Reply:
That config works for me. (It whines if it can't find redis though).